src/main.rs

```diff
@@ -143,7 +144,25 @@ fn setup(args: &[String]) -> Setup {
             "Audio device to use. Use '?' to list options if using portaudio",
             "DEVICE",
         )
-        .optopt("", "mixer", "Mixer to use", "MIXER")
+        .optopt("", "mixer", "Mixer to use (Alsa or softmixer)", "MIXER")
+        .optopt(
+            "m",
+            "mixer-name",
```

Comment:
This is the name of the (simple) control to use. This option should reflect that and be called `mixer-control` (like MPD).